modules/code-editor/src/views/full/LiteEditor/index.tsx

```diff
@@ -130,6 +157,14 @@ export default class MinimalEditor extends React.Component<Props> {
     this.setState({ code: unwrapped })
   }
 
+  getEditableZone = () => {
+    const lines = this.editor.getValue().split('\n')
+    const startLine = lines.findIndex(x => x.includes('Your code starts')) + 2
```

Comment:
is there a constant we can use for those ? if not can we export this from the template file  and use it here. ?

Reply:
isn't it `END_COMMENT` ?